### PR TITLE
pkg/operator: expose more resources to must-gather

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@linux.com>

**- What I did**

added collection of master,worker MCP and CC

**- How to verify it**

run must-gather

**- Description for the changelog**

### CODE REVIEW

File: pkg/operator/status.go [hunk 1]
```diff
@@ -207,6 +207,9 @@ func (optr *Operator) initializeClusterOperator() (*configv1.ClusterOperator, er
 	// RelatedObjects are consumed by https://github.com/openshift/must-gather
 	co.Status.RelatedObjects = []configv1.ObjectReference{
 		{Resource: "namespaces", Name: "openshift-machine-config-operator"},
+		{Group: "machineconfiguration.openshift.io", Resource: "machineconfigpools", Name: "master"},
+		{Group: "machineconfiguration.openshift.io", Resource: "machineconfigpools", Name: "worker"},
```
Comment (cgwalters) [inline pkg/operator/status.go]: We also need to support custom pools right?

Comment (runcom) [reply pkg/operator/status.go]: I've already a follow up to dynamically collect and populate this - I'm running through some testing and will open it once ready

Comment (cgwalters) [reply pkg/operator/status.go]: OK, so land as is and then improve later?  Cool by me!

Comment (runcom) [reply pkg/operator/status.go]: yup, that's the plan, at the beginning I actually just thought about having `master` only here. The information we gather from this is questionable also, logs from controller is what we'll look at anyway (and we gather that)

**Comment by runcom** [comment]
I'll add the 4.1.z BZ to the resulting cherry-pick PR, not this (bug is https://bugzilla.redhat.com/show_bug.cgi?id=1717738)

**Pull request merged**